sdk/quantum/Azure.Quantum.Jobs/README.md

```diff
@@ -34,27 +21,43 @@ Include a section after the install command that details any requirements that m
 
 > You must have an [Azure subscription](https://azure.microsoft.com/free/), [Cosmos DB account](https://docs.microsoft.com/azure/cosmos-db/account-overview) (SQL API), and [Python 3.6+](https://www.python.org/downloads/) to use this package.
 
-### Authenticate the client
+### Authentication
+
+To authenticate with the service, the workspace will use [DefaultAzureCredential](https://docs.microsoft.com/en-us/dotnet/api/azure.identity.defaultazurecredential?view=azure-dotnet) internally. This will try different authentication mechanisms based on the environment (e.g. Environment Variables, ManagedIdentity, CachedTokens) and finally it will fallback to [InteractiveBrowserCredential](https://docs.microsoft.com/en-us/dotnet/api/azure.identity.interactivebrowsercredential?view=azure-dotnet).
 
-If your library requires authentication for use, such as for Azure services, include instructions and example code needed for initializing and authenticating.
+Workspace will also allow the user to override the above behavior by passing their own [TokenCredential](https://docs.microsoft.com/en-us/dotnet/api/azure.core.tokencredential?view=azure-dotnet).
 
-For example, include details on obtaining an account key and endpoint URI, setting environment variables for each, and initializing the client object.
+In case the user already has an access token (using MSAL manually or even an external library), we will add an overload that accepts the access token string.
```

Comment:
I'd remove this because it doesn't help users get started if you're talking about future plans and we're looking at an OnBehalfOfToken/StaticToken in Azure.Identity that would be preferable to another overload.